### DISCUSSION TITLE
SentryPeer TCP on port 5060?

### DISCUSSION THREAD

**Discussion by ghenry** [telekom-security/tpotce#1215] [Q&A] [answered]
Hi all,

Where can I check if TCP port 5060 is exposed on T-Pot? I can see UDP is:
```
./tests/tools/sipexer 192.168.100.140:5060

SIP/2.0 200 OK
Via: SIP/2.0/UDP 127.0.0.1:56940
Call-ID: 1179563087@127.0.0.1
From: <sip:sipsak@127.0.0.1>;tag=464eb44f
To: <sip:asterisk@127.0.0.1>;tag=z9hG4bK.1c882828
CSeq: 1 OPTIONS
Accept: application/sdp, application/dialog-info+xml, application/simple-message-summary, application/xpidf+xml, application/cpim-pidf+xml, application/pidf+xml, application/pidf+xml, application/dialog-info+xml, application/simple-message-summary, message/sipfrag;version=2.0
Allow: OPTIONS, SUBSCRIBE, NOTIFY, PUBLISH, INVITE, ACK, BYE, CANCEL, UPDATE, PRACK, REGISTER, REFER, MESSAGE
Supported: 100rel, timer, replaces, norefersub
Accept-Encoding: text/plain
Accept-Language: en
Server: FPBX-14.0.16.11(14.7.8)
Content-Length:  0

localhost ~/src/sentrypeer [main*]$ ./tests/tools/sipexer tcp:192.168.100.140:5060

[error] [sipexer.go:1710] main.SIPExerSendTCP(): error: dial tcp4 192.168.100.140:5060: connect: connection refused
```

Port 5061 udp seems to be from honeytrap.

Thanks.

Thanks.

**Comment by ghenry** [comment, marked as answer]
Here would be good to have a tcp entry too :-)

https://github.com/telekom-security/tpotce/blob/2641d1e74306fafa9c43f08fb6927e3d9781c936/docker/sentrypeer/docker-compose.yml#L25

I'm still working on SIP TLS on port 5061. One for over Christmas.

**Reply by t3chn0m4g3** [reply]
This is just for testing. The T-Pot configuration can be found in `/opt/tpot/etc/tpot.yml` which is a link to one of the [compose](https://github.com/telekom-security/tpotce/blob/2641d1e74306fafa9c43f08fb6927e3d9781c936/etc/compose/standard.yml#L373-L391) files.
Is TCP fully supported now?

**Reply by ghenry** [reply]
Cool. Saw that on my local T-Pot install to confirm. Yep, for a while now.

Since May - https://github.com/SentryPeer/SentryPeer/releases/tag/v1.4.1

We're on v2.0.0 - https://github.com/SentryPeer/SentryPeer/releases/tag/v2.0.0

Thanks.

**Reply by t3chn0m4g3** [reply]
Thanks, I will be adjusting the config files accordingly.

**Comment by ghenry** [comment]
Also, if no p2p mode enable, you can delete:

https://github.com/telekom-security/tpotce/blob/2641d1e74306fafa9c43f08fb6927e3d9781c936/docker/sentrypeer/docker-compose.yml#L24

Thanks :-)
